controllers/admin/AdminStatsTabController.php

```diff
@@ -23,12 +23,12 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-abstract class AdminStatsTabControllerCore extends AdminPreferencesControllerCore
+abstract class AdminStatsTabControllerCore extends AdminController
```

Comment:
@matks, this seemed to not use anything from that controller, it might be just extended by mistake. I tried Stats page after this change and it works as before. So after all, i think we are able to delete AdminPreferencesController

Reply:
@zuk3975 Thanks for checking ! We have QA manual and automated tests to make sure we broke nothing, so let's do this 🚀 🎸